4.8/Dockerfile

```diff
@@ -6,15 +6,19 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
+    # Current keys from https://github.com/nodejs/node#release-team
```

Comment:
Ok, I understand. I'll update with the full URL.

Do we agree on still having `Current keys` and `Older keys` separation?

If you don't and are sure that this is for the better I understand and will return back the keys to where they were before and reduce the diff.